src/Drawer/index.js

```diff
@@ -31,10 +32,20 @@ const Drawer = ({
   const navRef = useRef(null);
 
   const isTransitioning = useMountTransition(isOpen, 300);
+  const { m } = useBreakpoints();
   const isHorizontal = position === "bottom" || position === "top";
+  const isVerticalMobileDrawer = (!m) && !isHorizontal;
+
   // The depth is how far the drawer opens out, but the CSS prop depends
   // on whether the Drawer is vertical or not
-  const depthStyle = isHorizontal ? { height: depth } : { width: depth };
+  const depthStyle = {
+    width: isHorizontal ? "auto" : depth,
+    height: isHorizontal ? depth : "100%",
```

Comment:
I _think_ this need to be `100%` - with `auto`, the drawer was doing this
![Screenshot 2024-03-28 at 1 34 23 PM](https://github.com/narmi/design_system/assets/50111807/c59a88d1-331e-498a-b6be-9a497c835279)

Reply:
I don't doubt I did something wrong here...

Would this do it?

```
const depthStyle = isHorizontal ? { height: depth } : { width: depth };

if (isVerticalMobileDrawer) {
   depthStyle.width = "100%";
}
```

Reply:
> I don't doubt I did something wrong here...
> 
> Would this do it?
> 
> ```
> const depthStyle = isHorizontal ? { height: depth } : { width: depth };
> 
> if (isVerticalMobileDrawer) {
>    depthStyle.width = "100%";
> }
> ```
> 
> I'm avoiding a nested ternary. We get a reasonable default style object, but if the mobile condition is true, we always set width to 100

Appears to have done the trick!